### ISSUE TITLE
A11y_PowerShellCore_Editor_Installation_ScreenReader: Jaws is not announcing the control type of the actionable elements which is "Hyperlink" when navigating using tab key.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode#127059]
### Actual:

Jaws is not announcing the control type of the actionable elements which is "Hyperlink" when navigating using tab key.

Same issue is observed with NVDA.

### Expected:

Jaws should announce the control type of the actionable elements which is "Hyperlinks" when navigating using tab key.

### Environment Details:
Application Name: PowerShell Core
Windows Version: Win10 

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#Win10;#DesktopApp;#Keyboard;#WCAG1.3.1;#PowerShell Core;
### 
Repro Steps:

1. Open Visual studio Application.
2. Tab to "Extension" button of activity bar.
3. Enter "PowerShell" in "Search Extension" text field.
4. Hit enter on install button it will install the power shell in vscode.
5. Tab till "Repository" using tab key and start jaws
6. Observe jaws announcement if it is announcing the control type of the actionable elements which is "Hyperlink" when navigating using tab key.
### 
User Impact:
Screen reader users will not be able to understand the correct purpose of the control

"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Comment by Shivamd1** [comment]
Logged external

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
